include/Photo.php

```diff
@@ -26,7 +26,7 @@ class Photo {
 	/**
 	 * @brief supported mimetypes and corresponding file extensions
 	 */
-	static function supportedTypes() {
+	static public function supportedTypes() {
```

Comment:
The usual order of keywords would be `public static function`.

Reply:
@MrPetovan Done